src/sentry/api/endpoints/organization_metrics_query.py

```diff
@@ -166,13 +168,31 @@ def post(self, request: Request, organization: Organization) -> Response:
                 },
             )
 
+            if all(
+                features.has("projects:use-eap-spans-for-metrics-explorer", project)
+                for project in projects
+            ):
+                if len(mql_queries) == 1 and "a" in mql_queries[0].sub_queries:
+                    subquery = mql_queries[0].sub_queries["a"]
+                    if "d:eap/" in subquery.mql:
+                        results = mql_eap_bridge.make_eap_request(
```

Comment:
Do you want to handle this with a try...except so we can fail over to the main pipeline?

Reply:
main pipeline would always fail with the 'eap' namespace, doesn't really add anything